### PR TITLE
Nuke v0.6.7 Examples

### PR DESCRIPTION
**Pull request by tristen**
This commit:
- Redirects any v0.6.7 examples to `/example/v1.0.0/`
- Removes liquid conditioning in templates for them
- Drops any assets dependent on them

### CODE REVIEW

File: _posts/examples/v1.0.0/0100-01-01-leaflet-minimap.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 version: v1.0.0
 title: Leaflet-MiniMap
 description: Using the Leaflet-MiniMap plugin
+tags:
```
Comment (tmcw) [inline _posts/examples/v1.0.0/0100-01-01-leaflet-minimap.html:7]: what's up with these unrelated changes?

Comment (tristen) [inline _posts/examples/v1.0.0/0100-01-01-leaflet-minimap.html:7]: I fixed this in `mb-pages` and didn't merge. Will sync up with `mb-pages`
